README.md

```diff
@@ -127,48 +131,56 @@ IAMユーザには以下の認可ポリシーが必要です。
 
 ## Usage
 
-`AwsSsmEnv#load`に渡すオプションの説明です。
+A description of the options passed to `AwsSsmEnv#load`.
 
 ### decryption: [Boolean]
 
-SecureStringのパラメータを復号化するかどうかを表すフラグ。  
-`true`を指定した場合は取得したSecureStringパラメータの値は復号化されている。  
-`false`の場合は暗号化されたまた環境変数値として設定される。  
-なお、このためのgemなのでデフォルトは`true`(復号化する)。
+Flag indicating whether to decrypt SecureString parameters.  
+If `true` is specified, the value of the acquired SecureString parameter is decrypted.  
+In case of `false` it is set as encrypted and environment variable value.  
+Since it is a gem for this, the default is `true` (decrypt).
 
 ### overwrite: [Boolean]
 
-すでに設定されている環境変数を上書きするかどうかを指定する。  
-`true`を指定した場合、環境変数が設定されていても取得したパラメータ値で上書きする。  
-`false`を指定した場合はすでに設定されている環境変数を上書きしない。  
-デフォルトは`false`(上書きしない)。  
-なお、`AwsSsmEnv#load!`を実行した場合、このフラグは自動的に`true`になる。
+Specify whether to overwrite an already set environment variable.  
+If `true` is specified, even if the environment variable is set,
+it overwrites it with the acquired parameter value.  
+If `false` is specified, do not overwrite already set environment variables.  
+The default is `false` (do not overwrite).  
+If you invoke `AwsSsmEnv#load!`, This flag will automatically be set to `true`.
 
 ### client: [Aws::SSM::Client]
 
-`Aws::SSM::Client`のインスタンスを指定する。  
-すでに生成済みのインスタンスがある場合にそれを設定するためのオプション。  
-生成済みのインスタンスがない場合は`ssm_client_args`を利用する。
+Specify an instance of `Aws::SSM::Client`.  
+An option to set it if there are already created instances.  
+If there are no instances already created use `ssm_client_args`.
 
 ### ssm_client_args: [Hash]
 
-`Aws::SSM::Client`のコンストラクタに渡すハッシュを指定する。  
-指定しなかった場合は引数なしで`Aws::SSM::Client.new`が呼ばれる。  
-環境変数やEC2インスタンスプロファイルによる認証情報を利用する場合は不要。
+Specify a hash to pass to the constructor of `Aws::SSM::Client`.  
+If not specified, `Aws::SSM::Client#new` is called with an empty argument.  
+It is unnecessary when using environment variable
+or authentication information by `EC2 InstanceProfile`.
 
 ### fetch: [Symbol, AwsSsmEnv::Fetcher, Object]
 
-パラメータ取得方法を指定する。  
-指定可能な値は`:path`, `:begins_with`または`AwsSsmEnv::Fetcher`を実装したクラスのインスタンス、`each`メソッドを
-持ったクラスのインスタンスのいずれか。  
-何も指定されていない場合は`:path`として扱われるが、後述の`begins_with`が指定されていた場合は自動的に`:begins_with`となる。
+Specify parameter fetch strategy.  
+Possible values are `:path`,`:begins_with`
+or an instance of a class that implements `AwsSsmEnv::Fetcher`,
+or an instance of a class with a `each` method.  
+If nothing is specified, it is treated as `:path`,
+but if `begins_with` is specified later,
```

Comment:
This is mistranslation.

> but when `begins_with` (which is described later) is specified,

This single sentence is too complex.
It's better to split like this:

> If nothing is specified, it is treated as `:path`. But when `begins_with` (which is described later) is specified, it will automatically be `:begins_with`.